gazebo_ros2_control/src/gazebo_ros2_control_plugin.cpp

```diff
@@ -200,6 +200,35 @@ void GazeboRosControlPlugin::Load(gazebo::physics::ModelPtr parent, sdf::Element
   // So we have to parse the plugin file manually and set it to the node's context.
   auto rcl_context = impl_->model_nh_->get_node_base_interface()->get_context()->get_rcl_context();
   std::vector<std::string> arguments = {"--ros-args", "--params-file", impl_->param_file_.c_str()};
+  if (sdf->HasElement("ros")) {
+    sdf = sdf->GetElement("ros");
+  }
```

Comment:
This will allow the following

```xml
<plugin>
  <namespace></namespace>
  <remapping></remapping>
  ...
</plugin>
```

and 

```xml
<plugin>
  <ros>
    <namespace></namespace>
    <remapping></remapping>
    ...
 </ros>
</plugin>
```

I think we should only go with the second option